src/components/Formatters/useCurrencyFormatter.tsx

```diff
@@ -0,0 +1,18 @@
+export enum CurrencypPosition {
+  BACK = 'de-DE',
+  FRONT = 'ja-JP',
+}
+
+export const useCurrencyFormatter = (
+  options?: Intl.NumberFormatOptions & { currencyFormat?: CurrencypPosition }
+) => {
+  const formatter = new Intl.NumberFormat(
+    options?.currencyFormat ?? CurrencypPosition.FRONT,
+    {
+      style: options?.style ?? 'currency',
+      currency: options?.currency ?? 'USD',
+      ...options,
```

Comment:
Can you also add these options as defaults?:
```tsx
  maximumFractionDigits: 0, 
  minimumFractionDigits: 0, 
```
We don't want to show cents by default.